cluster-autoscaler/FAQ.md

```diff
@@ -304,6 +305,112 @@ It can be added to (or removed from) a node using kubectl:
 kubectl annotate node <nodename> cluster-autoscaler.kubernetes.io/scale-down-disabled=true
 ```
 
+### How can I configure overprovisioning with Cluster Autoscaler?
+
+Below solution works since version 1.1 (to be shipped with Kubernetes 1.9).
+
+Overprovisioning can be configured by creating deployment which:
+ * reserves resources proportional to the number of cores in the cluster (i.e 200 millicores per core),
+ * frees resources when they are needed to schedule new pods,
+ * forces Cluster Autoscaler to scale up the cluster when there is not enough free resoures.
```

Comment:
still forces CA to scale up the cluster when there is not enough free resources to get the low priority pause pods scheduled.

Reply:
done